bindings/python/iota_sdk/client/client.py

```diff
@@ -406,7 +406,7 @@ def build_and_post_block(self,
             End of the input range
         inputs : Array of Inputs
             Inputs to use
-        output : SendAmountParams
+        output : SendParams
```

Comment:
🤔

Reply:
I changed the function argument name to `params` as well.